src/components/pages/collectRecordFormPages/BleachingForm/BleachingTransectInputs.js

```diff
@@ -149,6 +157,23 @@ const BleachingTransectInputs = ({
       <InputWrapper>
         <H2>Quadrat</H2>
 
+        <InputWithLabelAndValidation
+          disabled
+          label="Number"
+          id="number"
+          testId="number"
+          type="number"
+          ignoreNonObservationFieldValidations={() => {
+            ignoreNonObservationFieldValidations({ validationPath: NUMBER_VALIDATION_PATH })
+          }}
+          resetNonObservationFieldValidations={() => {
+            resetNonObservationFieldValidations({ validationPath: NUMBER_VALIDATION_PATH })
+          }}
+          {...labelValidationProperties}
+          onBlur={formik.handleBlur}
+          value={formik.values.number}
+          onChange={handleNumberChange}
+        />
```

Comment:
I think we can get rid of all the props other than:
- disabled
- id
- testId
- type
-value
-lable

Reply:
I would prefer to leave some of these - we are using `disabled`. also` id` should be there for html context and screen readers. 

not sure how` testId` is used, and it is not a required prop so may its fine to remove this . `type `doesn't seem to be listed in the `propTypes` for  `InputWithLabelAndValidation`

Reply:
I forgot to say, conditional approval on minimizing based on comments. :)